jobs/maintenance/buildvm_backup/build.groovy

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env groovy
+
+buildlib = load("pipeline-scripts/buildlib.groovy")
+commonlib = buildlib.commonlib
+
+// RPMS can periodically be synced with https://unix.stackexchange.com/a/189
+
+backupPlan = [
+    srcHost: 'buildvm.openshift.eng.bos.redhat.com',
+    destHost: 'buildvm2.openshift.eng.bos.redhat.com',
+    backupPath: '/mnt/workspace/backups/buildvm' // must exist on both src and dest host
+    files: [
+        '/etc/sysconfig/jenkins',  // config file for jenkins server
+        '/etc/sysconfig/docker', // insecure registries
+        '/etc', // for reference
+
+        // keys for accessing mirror repositories
+        '/var/lib/yum/client-cert.pem',
+        '/var/lib/yum/client-key.pem',
+
+        // /home/jenkins resolves to nfs share, so nothing needs to be copied
+
+        '/home/jenkins/.ssh',
+        '/home/jenkins/.config',
+        '/home/jenkins/.ssh',
+        '/home/jenkins/bin',
+        '/home/jenkins/.gem',
+        '/home/jenkins/.aws',
+        '/home/jenkins/.docker',
+        '/home/jenkins/.kube',
+        '/home/jenkins/credentials',
+        '/home/jenkins/kubeconfigs',
+        '/home/jenkins/go',
+
+        // Jenkins war
+        '/usr/lib/jenkins',
+
+        // Jenkins configuration, plugins, etc
+        '/mnt/nfs/jenkins_home',
```

Comment:
Not so certain about this one. I'm running a `du` on that right now and it has been running for about a minute. We might need to clean up that location.

Reply:
Yeah it's still running.

Reply:
Hm. I know what we need here. To run the actual jenkins backup script. It picks out the useful stuff. `/root/jenkins-backup.sh` has it.

> usage: jenkins-backup.sh /path/to/jenkins_home archive.tar.gz

I invoke it like this

> ./jenkins-backup.sh /mnt/nfs/jenkins_home `pwd`/2019-09-03-1.tar.gz

That omits log files. Which in theory won't be a huge issue because we're going to be jamming those into splunk soon anyway.